### PR TITLE
git-show: various tweaks

### PR DESCRIPTION
**Pull request by waldyrious**
- adjust punctuation & spacing in main description
- adjust descriptions ("changes" --> "information", and explicitly list which info is returned in the first example)
- use a concrete example for the tilde notation
- add abbreviated example

Follow-up of #1331.

### CODE REVIEW

**Comment by waldyrious** [comment]
well, since @mention-bot seems to be hit-and-miss lately, I'll ping @alefir manually.

**Comment by alefir** [comment]
On line 13, I'd change 'associated to' to 'associated with'.

Besides that, looks good to me.

File: pages/common/git-show.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
 # git show
 
-> Show various types of git objects(commits, tags etc.).
+> Show various types of git objects (commits, tags, etc.).
 
-- Show changes made in the latest commit:
+- Show information about the latest commit (message, differences, and other metadata):
```
Comment (agnivade) [inline pages/common/git-show.md]: I think "changes" is a better word than "differences".

**Comment by waldyrious** [comment]
Thanks for the comments. I agree with both of your proposals, and have implemented them.

**Pull request merged**